src/servletTutorial/servlets/EditBookServlet.java

```diff
@@ -0,0 +1,79 @@
+package servletTutorial.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import servletTutorial.manage.BookManagement;
+import servletTutorial.validation.BookValidation;
+
+/**
+ * Servlet implementation class EditBookServlet
+ */
+@WebServlet("/EditBookServlet")
+public class EditBookServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * @see HttpServlet#HttpServlet()
+	 */
+	public EditBookServlet() {
+		super();
+		// TODO Auto-generated constructor stub
+	}
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse
+	 *      response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		// TODO Auto-generated method stub
+		int bookId = Integer.parseInt(request.getParameter("id"));
+
+		request.setAttribute("data", BookManagement.showEditBook(bookId));
```

Comment:
Method `showEditBook` should change to `getBook(bookId)`. It will help easy to understand.